docs/guidelines.md

```diff
@@ -309,3 +309,42 @@ For example, don't assign a feature to both `css` and `fonts`, since `css` is th
 
 Do assign features to groups when there's an opportunity for future feature composition (see [#971](https://github.com/web-platform-dx/web-features/issues/971)).
 For example, several features for the JavaScript `Array` interface are members of the `array` group.
+
+## Discouraged
+
+Rarely, the developers should not use a platform feature because it's the consensus of relevant stakeholders (i.e., a standards body and implementers), even if that feature is still implemented in browsers.
+Mark a feature as discouraged when:
+
+- The specification adopts clear language directing developers to stop using that feature or to prefer to an alternative.
+  This is often through terms like "deprecated", "obsolete", "legacy", or "non-standard."
```

Comment:
I would omit the "non-standard" part here. There are many non-standard (yet) features that are not discouraged, per se.

Reply:
OK, I've removed this with fc94af6beac7a6f53ed19977961ddf452f14a9bc. I think it's _true_—particularly when a spec absorbs some but not all functionality from a non-standard predecessor—but it probably doesn't add clarity here (without more explanation).